io.github.cudatext.CudaText-Qt.yml

```diff
@@ -0,0 +1,186 @@
+        sha256: 5adac308a5534b6a76446d8311fc340747cbb7edeaacfe6b651493ff3fe31e83
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://downloads.sourceforge.net/project/freepascal/Linux/3.2.2/fpc-3.2.2.aarch64-linux.tar
+        sha256: b39470f9b6b5b82f50fc8680a5da37d2834f2129c65c24c5628a80894d565451
+      - type: archive
+        url: https://downloads.sourceforge.net/project/freepascal/Source/3.2.2/fpcbuild-3.2.2.tar.gz
+        sha256: 85ef993043bb83f999e2212f1bca766eb71f6f973d362e2290475dbaaf50161f
+      - type: patch
+        options: ['-d', 'fpcsrc']
+        path: fpc-3.2.0--glibc-2.34.patch
+    buildsystem: simple
+    build-options:
+      env:
+        - fpcver=3.2.2
+        - PREFIX=/app/lib/sdk/freepascal
+      arch:
+        aarch64:
+          env: [PPNAME=ppca64]
+        x86_64:
+          env: [PPNAME=ppcx64]
+    build-commands:
+      # install src
+      - mkdir -p ${PREFIX} && cp -r fpcsrc ${PREFIX}/
+      # tar x fpcpre
+      - mkdir fpcpre
+      - tar -xf binary.*.tar && tar -C fpcpre -xf base.*.tar.gz
+      - rm *.tar.gz *.tar
+      # compile self
+      - make -C fpcsrc compiler_cycle PP=`pwd`/fpcpre/lib/fpc/${fpcver}/${PPNAME}
+      # compile rtl with new pp
+      - make -C fpcsrc rtl_clean rtl_smart PP=`pwd`/fpcsrc/compiler/${PPNAME}
+      # compile others with new pp,rtl
+      - make -C fpcsrc packages_smart utils_all PP=`pwd`/fpcsrc/compiler/${PPNAME}
+      # install
+      - make -C fpcsrc compiler_distinstall rtl_distinstall packages_distinstall utils_distinstall
+        INSTALL_PREFIX=${PREFIX}
+        PP=`pwd`/fpcsrc/compiler/${PPNAME}
+        FPCMAKE=`pwd`/fpcsrc/utils/fpcm/bin/${FLATPAK_ARCH}-linux/fpcmake
+      # make cfg, add lib path, and link ppc
+      -  ${PREFIX}/lib/fpc/${fpcver}/samplecfg ${PREFIX}/lib/fpc/${fpcver} ${PREFIX}/etc
+      -  echo '-Fl/app/lib' >> ${PREFIX}/etc/fpc.cfg
+      -  ln -s ../lib/fpc/${fpcver}/${PPNAME} ${PREFIX}/bin/${PPNAME}
+      # fix not found ppcaarch64
+      -  ln -s ppca64 ${PREFIX}/bin/ppcaarch64
+
+  - name: lazarus
+    sources:
+      - type: archive
+        url: https://gitlab.com/freepascal.org/lazarus/lazarus/-/archive/lazarus_3_0/lazarus-lazarus_3_0.tar.gz
+        sha256: b16f1f6339d43bef8a1ecedd854a2065e60136fdca75149c6321f5c5b54f49dd
+    buildsystem: simple
+    build-options:
+      append-path: /app/lib/sdk/freepascal/bin
+      env:
+        - INSTALL_PREFIX=/app/lib/sdk/freepascal
+    build-commands:
+      - find . -type f -iname makefile.fpc -exec fpcmake -Tall '{}' ';'
+      - make lazbuild OPT='-gl'
+      - touch lazarus startlazarus
+      - make install
+      - sed -i -e "s#__LAZARUSDIR__#$LAZARUSDIR/#" -e "s#__FPCSRCDIR__#$FPCDIR/#" -e "s#/app/bin/fpc#$INSTALL_PREFIX/bin/fpc#"
+        tools/install/linux/environmentoptions.xml
+      - install -Dm644 -t $LAZARUSDIR tools/install/linux/environmentoptions.xml
+
+  - name: qt6pas
+    buildsystem: simple
```

Comment:
should this be `buildsystem: qmake`?

Reply:
I think it needs to be `simple` as we need the sed command to fix the target.path parameter.